libs/cart/testing/src/factories/magento/product.factory.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import * as faker from 'faker/locale/en_US';
+
+import { MagentoProduct, MagentoMoney } from '@daffodil/cart';
+
+import { DaffModelFactory } from '@daffodil/core/testing';
+import { MagentoMoneyFactory } from './money.factory';
+
+export class MockMagentoProduct implements MagentoProduct {
+  id = faker.random.number(1000);
+  image = {
+    label: faker.random.words(3),
+    url: faker.image.imageUrl()
+  };
+  manufacturer = faker.random.number(1000);
+  name = faker.random.word();
+  description = faker.random.words(5);
+  price_range = {
+    maximum_price: this.money(),
+    minumum_price: this.money()
+  };
+  sku = faker.random.alphaNumeric(16);
+
+  private money(): MagentoMoney {
+    return (new MagentoMoneyFactory()).create()
+  }
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class MagentoProductFactory extends DaffModelFactory<MagentoProduct> {
```

Comment:
I think this and the product model in @daffodil/cart is going to be a problem. Right now, the one in @daffodil/product is called `ProductNode`, but it should really be called `MagentoProduct`. It's just weird because the cart package is dependent on product package. Maybe this one could be called `MagentoCartItemProduct` or something.

Reply:
Well really they are they same backend model. I think ideally we would have a product model that resembled the actual [Magento `ProductInterface`](https://devdocs.magento.com/guides/v2.3/graphql/product/product-interface.html) that both the cart and product drivers could use.

Reply:
Sounds good to me. Want to remove the product factory here, then do a quick PR for adding it to the product package?

Reply:
Or do you need to update the interface too?

Reply:
I don't need to change the interface, `ProductNode` has everything that I need for `DaffCartItem`. I'll submit that PR in a sec.

Reply:
`MagentoProductFactory` has been removed from this PR.